admin/plugins.php

```diff
@@ -0,0 +1,395 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Get info about all the standalone plugins and their status.
+ *
+ * @since n.e.x.t
+ *
+ * @return array of plugins info.
+ */
+function perflab_get_standalone_plugins() {
+	$managed_standalone_plugins = array(
+		'webp-uploads',
+		'performant-translations',
+		'dominant-color-images',
+	);
+
+	return $managed_standalone_plugins;
+}
+
+/**
+ * Renders plugin UI for managing standalone plugins within PL Settings screen.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_plugins_ui() {
+	require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+	require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+	$standalone_plugins = perflab_get_standalone_plugins();
```

Comment:
At some point, when we merge the standalone plugin into the core, the list of standalone plugins should be an empty array. Therefore, we should implement a check that prevents displaying the Plugin UI if there are no standalone plugins available.

Reply:
@mukeshpanchal27 this has been addressed in https://github.com/WordPress/performance/pull/864/commits/b2995cd021bafae4e0d12b7d28e4ab8601eb30b3, thank you